### PR TITLE
[Feature] Add 500 page

### PR DESCRIPTION
**Pull request by gertie-sheshe**
## Description

Adds a 500 page.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: src/pages/500.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+}
+
+export async function getStaticProps({ locale }) {
+  const backend = backendFn();
+  const site = await backend.sites().current;
+  const { navigation } = site;
+  const wpApi = wp();
+  const page = await wpApi.pages({ slug: "error", locale }).first;
+  const posts = await wpApi.pages({ slug: "analysis-articles", locale }).posts;
+  const articles = posts?.slice(0, 4) || null;
```
Comment (kilemensi) [inline src/pages/500.js:19]: I think we need to check if site has articles enabled and if yes, pull them from the `backend`

**Review by kilemensi — COMMENTED** [review]
👍🏽

File: src/pages/500.js (hunk 1 above)
Comment (kilemensi) [reply src/pages/500.js:19]: @gertie-sheshe ?

Comment (gertie-sheshe) [reply src/pages/500.js:19]: @kilemensi What do you mean by `site has articles enabled`

**Review by KobbyMmo — APPROVED** [review]

**Pull request merged**